setup.cfg

```diff
@@ -17,6 +17,7 @@ python_requires = >=3.6
 setup_requires = setuptools_scm
 install_requires =
     astropy>=5.2 # astropy with s3fs support
+    astroquery>=0.4.6 # for testing
```

Comment:
if this is only needed for testing, can it be moved into the test extra section below?

Reply:
Sure thing! I'll address these comments when I fix the CI. Thanks!